scripts/setup-centos8.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -efx -o pipefail
+# Some of the packages must be build with the same compiler flags
+# so that some low level types are the same size. Also, disable warnings.
+SCRIPTDIR=$(dirname "${BASH_SOURCE[0]}")
+source $SCRIPTDIR/setup-helper-functions.sh
+CPU_TARGET="${CPU_TARGET:-avx}"
+NPROC=$(getconf _NPROCESSORS_ONLN)
+export CFLAGS=$(get_cxx_flags $CPU_TARGET)  # Used by LZO.
+export CXXFLAGS=$CFLAGS  # Used by boost.
+export CPPFLAGS=$CFLAGS  # Used by LZO.
+
+function dnf_install {
+  dnf install -y -q --setopt=install_weak_deps=False "$@"
+}
+
+dnf_install epel-release dnf-plugins-core # For ccache, ninja
+dnf config-manager --set-enabled powertools
+dnf_install ninja-build ccache gcc-toolset-9 git wget which libevent-devel \
+  openssl-devel re2-devel libzstd-devel lz4-devel double-conversion-devel \
+  libdwarf-devel curl-devel
+
+dnf remove -y gflags
+
+# Required for Thrift
+dnf_install autoconf automake libtool bison flex python3
+
+# install sphinx for doc gen
+pip3 install sphinx sphinx-tabs breathe sphinx_rtd_theme
```

Comment:
should we create a virtualenv instead of installing directly?

Reply:
Do you mean conda env ?

Reply:
I mean a Python virtualenv so we don't install system dependencies on users that are using this script on their local hosts:
```
python -m venv velox-venv
source velox-venv/bin/activate
```
Conda could also work but I think that virtualenv is enough and we don't have to install conda, etcetera.

Reply:
Python libs are not useful for code compiling.
Just removed